src/app/api/admins/route.js

```diff
@@ -0,0 +1,121 @@
+import { getServerSession } from "next-auth/next";
+import { NextResponse } from "next/server";
+import { authOptions } from "../auth/[...nextauth]/route";
+import { db } from "../../../../firebase";
+import {
+  doc,
+  updateDoc,
+  collection,
+  getDocs,
+  query,
+  where,
+  arrayRemove,
+  deleteField,
+} from "firebase/firestore";
+
+export async function POST(req) {
+  const res = NextResponse;
+  const { discord, affiliations, position } = await req.json();
+  const session = await getServerSession(authOptions);
+
+  if (session) {
+    try {
+      await updateDoc(doc(db, "users", session.user.id), {
+        discord: discord,
+        affiliations: affiliations,
+        position: position,
+        "status.admins": "pending",
```

Comment:
the page is /admin, we should change that to admins then, can u make the page change to /admins, make sure this is also reflected in folder names and such